packages/backend/src/open_payments/quote/service.test.ts

```diff
@@ -227,6 +233,7 @@ describe('QuoteService', (): void => {
         beforeEach(async (): Promise<void> => {
           incomingPayment = await createIncomingPayment(deps, {
             paymentPointerId: receivingPaymentPointer.id,
+            grantId: grant.id,
```

Comment:
I don't think this file needs to be updated now that `grantId` is optional

Reply:
I thought so, too, but then I realized that when querying the incoming payment, we proxy the request to the endpoint (to make it https) and add the `testAccessToken` (see [here](https://github.com/interledger/rafiki/blob/68115bb912ed75bb8d72bb542641df9f5dcff1d7/packages/backend/src/tests/app.ts#L86)-L98). Hence, there is a `grant` and a `clientId` associated with that incoming payment. Since the grant only has access type `read`, it will filter on that `clientId`. If we don't add the `grantId` in the test, the query fails.

Reply:
Yes we could do that, also. I just added the change to make sure that 'read-all` includes `read` (same for list).